tests/race-with-remove-dir.rs

```diff
@@ -10,14 +10,16 @@ fn test_race_with_remove_dir() {
 
     {
         let tmpdir = tmpdir.path().to_path_buf();
-        thread::spawn(move || {
-            let mut watcher = notify::recommended_watcher(move |result| {
-                eprintln!("received event: {:?}", result);
-            })
-            .unwrap();
+        let _ = thread::Builder::new()
+            .name("notify-rs test-race-with-remove-dir".to_string())
```

Comment:
I'm not sure we want thread names in tests, as it'll just add more code noise, but I guess its fine

Reply:
I had the same doubts - but I figured why not.